packages/pn-personafisica-webapp/src/api/notifications/__test__/Notifications.api.test.ts

```diff
@@ -31,50 +30,70 @@ import {
   NOTIFICATION_PAYMENT_INFO,
   NOTIFICATION_PAYMENT_URL,
 } from '../notifications.routes';
+import { mockApi } from '../../../__test__/test-utils';
 
 describe('Notifications api tests', () => {
+  let mock: MockAdapter;
   mockAuthentication();
+  afterEach(() => {
+    if (mock) {
+      mock.restore();
+      mock.reset();
+    }
+  });
 
   it('getReceivedNotifications', async () => {
-    const mock = new MockAdapter(apiClient);
-    mock
-      .onGet(
-        NOTIFICATIONS_LIST({
-          startDate: formatToTimezoneString(tenYearsAgo),
-          endDate: formatToTimezoneString(getNextDay(today)),
-        })
-      )
-      .reply(200, notificationsFromBe);
+    mockApi(apiClient, 'GET', NOTIFICATIONS_LIST({
+      startDate: formatToTimezoneString(tenYearsAgo),
+      endDate: formatToTimezoneString(getNextDay(today)),
+    }), 200, undefined, notificationsFromBe);
     const res = await NotificationsApi.getReceivedNotifications({
       startDate: formatToTimezoneString(tenYearsAgo),
       endDate: formatToTimezoneString(getNextDay(today)),
     });
     expect(res).toStrictEqual(notificationsToFe);
-    mock.reset();
-    mock.restore();
+  });
+
+  it('getReceivedNotifications with server error', async () => {
```

Comment:
error check is useless

Reply:
changes made into: [430db73](https://github.com/pagopa/pn-frontend/pull/938/commits/430db73dd4e6ee0215846db2b30db4717bdd1763)